README.md

```diff
@@ -1,69 +1,239 @@
-bem-walk
-========
+# bem-walk
 
-[![NPM Status][npm-img]][npm]
```

Comment:
Hey, everybody loves such badges. And they are very useful while searching new npm module.